flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -283,12 +284,13 @@ static void _msp_send_analog(struct msp_bridge *m)
 		FlightBatterySettingsGet(&batSettings);
 	}
 
-	// TODO:  Verify these with a board that has working power stuff
 	if (batSettings.VoltagePin != FLIGHTBATTERYSETTINGS_VOLTAGEPIN_NONE)
 		data.status.vbat = (uint8_t)lroundf(batState.Voltage * 10);
 
-	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE)
-		data.status.current = lroundf(batState.Current * 10);
+	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE) {
+		data.status.current = lroundf(batState.Current * 100);
```

Comment:
FWIW, when using variables that are scaled such that they are no longer using "normal" units, my personal preference is to see that in the variable name itself. So this could be data.status.current_100. There's a double advantage to this: 1) it's clear to any downstream users that this variable isn't containing a value that can be used unscaled and 2) if there are any pre-existing downstream users of the variable the compiler will fail until they're all refactored, giving the opportunity to make sure each and every scaling is correct.